api/app/signals/apps/dataset/sources/cbs.py

```diff
@@ -40,7 +31,10 @@ class CBSBoundariesLoader(AreaLoader):
 
     PROVIDES = DATASET_INFO.keys()
 
-    def __init__(self, type_string, directory):
+    def __init__(self, **options):
+        type_string = options['type_string']
+        directory = options['dir']
```

Comment:
Make this `self.directory = options['dir']`, no need for the temporary variable `directory`.

Reply:
check